hack/e2e-cleanup.sh

```diff
@@ -177,6 +177,19 @@ function iam_ra_cluster_name_tag_from_resource(){
     echo "$cluster_name"
 }
 
+# See above note about role tags
+function ssm_parameter_cluster_name_tag(){
+    id="$1"
+    >&2 echo "($(pwd)) \$ command aws ssm list-tags-for-resource --resource-id $id --resource-type Parameter --query "{Tags:TagList}" --output json" 
```

Comment:
remove?

Reply:
ive been keeping these in this script to make it easier to track whats going on, if you are trying to follow the logs.  The reason not all calls all this is that most of the calls `aws` is wrapped by `echo_and_run`, but for these where we check the error code, they are not so I have this manual log message.